mdx2/data.py

```
@@ -393,14 +522,45 @@ def chunk_slice_iterator(self):
                     yield (a1, a2, a3)
 
+    def iter_slabs(self):
+        """generator of ImageSeries objects, each containing a slab of images along the phi axis"""
+        for sl in self.chunk_slice_along_axis(0):
+            new_phi = self.phi[sl]
+            new_exposure_times = self.exposure_times[sl]
+            if isinstance(self.data, NXfield) and self.data._value is None and self.data._memfile is None:
+                # TODO: there might be instances where this has strange behavior, for instance if the NXfield was
+                # already saved to a file
+                new_data = self.data.__deepcopy__()
+                new_data.shape = (new_phi.size,) + new_data.shape[1:]
```

Ruff (0.14.8)

Consider `(new_phi.size, *new_data.shape[1:])` instead of concatenation

Replace with `(new_phi.size, *new_data.shape[1:])`

(RUF005)